Makefile

```diff
@@ -23,3 +23,27 @@ test:
 test-watch:
 	ptw
 
+################
+### LOCALNET ###
+################
+
+# Start an existing localnet or init a new one.
+localnet-start:
+		$(MAKE) -C localnet start
+
+# Configure the localnet keys and service account. Update .env to match.
+localnet-configure:
+		$(MAKE) -C localnet configure
+		@echo "Configuring .env for localnet"
+		cp .env.default .env
+		echo "REGEN_CHAIN_ID = \"$$(make -sC localnet localnet-chain-id)\"" >> .env
+		echo "REGEN_KEY_ADDRESS = \"$$(make -sC localnet keys-manager-address)\"" >> .env
```

Comment:
@wgardiner running `make localnet-configure` from the root of this repo copies `.env.default` to `.env` with the localnet-specific settings.